charts/all-in-one/templates/rudolf-service.yaml

```diff
@@ -38,8 +38,10 @@ spec:
                 secretKeyRef:
                   key: database-url
                   name: rudolf-service
+{{- if .Values.rudolfService.config.ncgMinter }}
```

Comment:
indent please 🙏

Reply:
When I test with `9c-internal/9c-network/values.yaml` it works well 🤔 Is there another way to test it?

Command:
```
helm template charts/all-in-one --values 9c-internal/9c-network/values.yaml > tmp.yaml
```

Rendered:
```yaml
# Source: 9c-network/templates/rudolf-service.yaml
apiVersion: apps/v1
kind: Deployment
metadata:
  labels:
    app: 9c-rudolf
    app.kubernetes.io/instance: release-name
  name: 9c-rudolf
  namespace: release-name
spec:
  replicas: 1
  selector:
    matchLabels:
      app: 9c-rudolf
  template:
    metadata:
      labels:
        app: 9c-rudolf
    spec:
      containers:
        - name: 9c-rudolf
          env:
            - name: DATABASE_URL
              valueFrom: 
                secretKeyRef:
                  key: database-url
                  name: rudolf-service
            - name: NCG_MINTER
              value: 0x47D082a115c63E7b58B1532d20E631538eaFADde
            - name: GQL_ENDPOINT
              value: https://9c-internal-rpc-1.nine-chronicles.com/graphql
            - name: NC_GRAPHQL_ENDPOINT
              value: https://9c-internal-rpc-1.nine-chronicles.com/graphql
            - name: AWS_KMS_KEY_ID
              value: 54436222-3b06-4ddb-b661-f2cd54456893
            - name: AWS_KMS_PUBLIC_KEY
              value: 04ff006e2434dc04000971395e5e47012e4ec7570dfbbb87a02e4b12d33ec0c6ec329fdba089f7b5bfce7b8cbcdf3f9e662fade6a63066a9b1e17429687fbdb9de
          image: planetariumhq/9c-rudolf:git-cda9ad1b4c5269a052a18ca3b0a3b94d48f51491
          ports:
            - containerPort: 3000
          resources:
            {}
      restartPolicy: Always
      serviceAccount: release-name-9c-rudolf-iam-role
      serviceAccountName: release-name-9c-rudolf-iam-role
```

Reply:
it doesnt affect the result and is not important, just recommended for good looking.